src/slider.jsx

```diff
@@ -44,26 +44,92 @@ let valueInRangePropType = (props, propName, componentName) => {
 const Slider = React.createClass({
 
   propTypes: {
+    /**
+     * The default value of the slider.
+     **/
     defaultValue: valueInRangePropType,
+
+    /**
+     * Describe the slider.
+     **/
     description: React.PropTypes.string,
+
+    /**
+     * Disables focus ripple if set to true.
+     **/
     disableFocusRipple: React.PropTypes.bool,
+
+    /**
+     * If true, the slider will not be interactable.
+     **/
     disabled: React.PropTypes.bool,
+
+    /**
+     * An error message for the slider.
+     **/
     error: React.PropTypes.string,
+
+    /**
+     * The maximum value the slider can slide to on
+     * a scale from 0 to 1 inclusive. Cannot be equal to min.
+     **/
     max: minMaxPropType,
+
+    /**
+     * The minimum value the slider can slide to on a scale
+     * from 0 to 1 inclusive. Cannot be equal to max.
+     **/
     min: minMaxPropType,
+
+    /**
+     * The name of the slider. Behaves like the name attribute
+     * of an input element.
+     **/
     name: React.PropTypes.string.isRequired,
```

Comment:
Why is this required?
For instance, it's not here `http://react-component.github.io/slider/`.